src/ast/mod.rs

```diff
@@ -1832,6 +1833,11 @@ pub enum Statement {
         name: ObjectName,
         representation: UserDefinedTypeRepresentation,
     },
+
+    Vacuum {
```

Comment:
Can you please add a documentation string here and a link to the reference documentation? https://www.postgresql.org/docs/current/sql-vacuum.html